Potential NPE in the Description class if the user doesn't configure any optional rows checks

```
Caused by: java.lang.NullPointerException
	at java.util.ArrayList.<init>(ArrayList.java:178)
	at org.jboss.tyr.model.yaml.Description.getOptionalRows(Description.java:40)
	at org.jboss.tyr.check.TemplateChecker.registerChecks(TemplateChecker.java:89)
	at org.jboss.tyr.check.TemplateChecker.init(TemplateChecker.java:51)
	at org.jboss.tyr.check.TemplateChecker_ClientProxy.init(TemplateChecker_ClientProxy.zig:157)
```

--- a/tyr-runner/src/main/java/org/jboss/tyr/model/yaml/Description.java
+++ b/tyr-runner/src/main/java/org/jboss/tyr/model/yaml/Description.java
@@ -23,10 +23,10 @@ import java.util.List;
 public class Description {
 
     @JsonProperty("required rows")
-    private List<RegexDefinition> requiredRows;
+    private List<RegexDefinition> requiredRows = new ArrayList<>();
 
     @JsonProperty("optional rows")
-    private List<OptionalPattern> optionalRows;
+    private List<OptionalPattern> optionalRows = new ArrayList<>();
 
     public List<RegexDefinition> getRequiredRows() {
         return new ArrayList<>(requiredRows);

